plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialLevel.java

```diff
@@ -51,7 +51,7 @@ public static void registerTags() {
         // @returns ElementTag(Number)
         // @group properties
         // @description
-        // Returns the maximum level for a Levelled material (like water, lava, and cauldrons), cake, beehives, and snow.
+        // Returns the maximum level for a Levelled material (like water, lava, cauldrons, cake, beehives, and snow).

```

Comment:
This is wrong. The sentence is `Levelled material, cake, beehives, and snow.` The parens indicate what a Levelled material is.